app/pages/lab/visibility.cjsx

```diff
@@ -169,6 +169,9 @@ module.exports = React.createClass
             {@state.workflows.map (workflow) =>
               setting = workflow.active
               stats_completeness_type = workflow.configuration.stats_completeness_type ? 'retirement'
+              stats_visible = !workflow.configuration.stats_hidden
+              if !workflow.active
+                stats_visible = !(workflow.configuration.stats_hidden == undefined) && stats_visible
```

Comment:
I'm not sure what this comparison does. My reading is: if `stats_hidden` is undefined, then `stats_visible` will be true and this comparison is redundant. If `stats_hidden` is defined then this becomes `false && !stats_hidden` which is always false.